lib/jekyll/theme.rb

```diff
@@ -62,11 +62,22 @@ def realpath_for(folder)
       # escape the theme root.
       # However, symlinks are allowed to point to other directories within the theme.
       Jekyll.sanitized_path(root, File.realpath(Jekyll.sanitized_path(root, folder.to_s)))
-    rescue Errno::ENOENT, Errno::EACCES, Errno::ELOOP
-      Jekyll.logger.warn "Invalid theme folder:", folder
+    rescue Errno::ENOENT, Errno::EACCES, Errno::ELOOP => e
+      handle_exception(e, folder)
       nil
     end
 
+    def handle_exception(err, folder)
```

Comment:
`log_realpath_exception` ?

Reply:
Yeah, that'd communicate the intent clearly.. :+1:

Reply:
@cetinajero Will you be able to push the changes right away..?

Reply:
For sure!